lib/final_destination.rb

```diff
@@ -44,9 +44,9 @@ def initialize(url, opts = nil)
     @opts[:max_redirects] ||= 5
     @opts[:lookup_ip] ||= lambda { |host| FinalDestination.lookup_ip(host) }
 
-    @ignored = @opts[:ignore_hostnames] || []
```

Comment:
Just making sure we checked through the plugins to ensure that this option isn't being used. Otherwise, we might break those plugins if they are not well tested.

Reply:
Actually after looking around things seem fine, but the bigger question I have is shouldn't I be able to trust our actions tests.yml plugin tests..?